specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-07-01/virtualMachineScaleSet.json

```diff
@@ -5318,6 +5356,10 @@
           "$ref": "#/definitions/PriorityMixPolicy",
           "description": "Specifies the desired targets for mixing Spot and Regular priority VMs within the same VMSS Flex instance."
         },
+        "skuProfile": {
```

Comment:
let's put this at the end of the property list for SDK generation to add new parameter at the end